.github/workflows/test-backend.yml

```
@@ -19,13 +19,21 @@ jobs:
+        uses: ./.github/actions/db-restore
         with:
-          network: ${{ vars.SUPABASE_NETWORK }}
           pull: ${{ !env.ACT }}
 
+      - name: Start supabase in background (while installing deps)
+        uses: ./.github/actions/supabase-start
+        with:
+          network: ${{ vars.SUPABASE_NETWORK }}
+          background: true
+
+      - uses: ./.github/actions/pnpm-install
```

actionlint (1.7.12)

[error] missing input "bryntum_access_token" which is required by action "pnpm-install" defined at "./.github/actions/pnpm-install". all required inputs are "bryntum_access_token"

(action)